third_party/nvfuser/csrc/scheduler/utils.cpp

```diff
@@ -233,6 +233,38 @@ void parallelizeAllLike(
     std::vector<TensorView*> selected_tvs,
     const std::unordered_set<ParallelType>& selected_parallel_types,
     bool propagate_padding) {
+#ifndef NDEBUG
+  {
+    std::stringstream ss;
+    ss << "{";
+    bool comma = false;
+    for (auto t : selected_tvs) {
+      if (comma)
+        ss << ", ";
```

Comment:
There's also `toDelimitedString` in utils.h, which you may be able to just use or extend.